src/dimension.h

```diff
@@ -24,6 +24,8 @@ typedef struct Dimension
 	DimensionType type;
 	AttrNumber	column_attno;
 	Oid			main_table_relid;
+	Oid			time_func;
+	Oid			time_func_rettype;
```

Comment:
There are only used if type is open? It needs a comment saying so, and what the values should be (if any) otherwise.